<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SwarmSettingsWidget</class>
 <widget class="QWidget" name="SwarmSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Swarm Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout_3">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Zone of Repulsion:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="zoneOfRepulsionSpinbox">
       <property name="maximum">
        <double>5000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Zone of Attraction:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="zoneOfAttractionSpinbox">
       <property name="maximum">
        <double>5000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Zone of Orientation:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="zoneOfOrientationSpinbox">
       <property name="maximum">
        <double>5000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Turning Rate:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="turningRateSpinbox">
       <property name="maximum">
        <double>100000.000000000000000</double>
       </property>
       <property name="value">
        <double>40.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Speed:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="speedSpinbox">
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="value">
        <double>3.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Field of view</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="alphaSpinbox">
       <property name="minimum">
        <double>2.000000000000000</double>
       </property>
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
       <property name="value">
        <double>270.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Number of Agents:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="numAgentsSpinbox">
       <property name="maximum">
        <number>500000</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="startResetButton">
       <property name="text">
        <string>Start/Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>zoneOfRepulsionSpinbox</tabstop>
  <tabstop>zoneOfAttractionSpinbox</tabstop>
  <tabstop>zoneOfOrientationSpinbox</tabstop>
  <tabstop>numAgentsSpinbox</tabstop>
  <tabstop>startResetButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
